<ui version="4.0" >
 <class>SoilDialog</class>
 <widget class="QDialog" name="SoilDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Soils</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>420</x>
     <y>590</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>590</y>
     <width>77</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text" >
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>590</y>
     <width>77</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text" >
    <string>Add</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="_soilTable" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>771</width>
     <height>281</height>
    </rect>
   </property>
   <property name="rowCount" >
    <number>0</number>
   </property>
   <property name="columnCount" >
    <number>7</number>
   </property>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>10</y>
     <width>53</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Soils</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_name" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>330</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>310</x>
     <y>330</y>
     <width>33</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_elevation" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>360</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>360</y>
     <width>135</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Top Layer Elevation [m]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>390</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Total porosity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_totalPorosity" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>390</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="_waterPorosity" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>420</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>420</y>
     <width>114</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Water filled porosity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_organicCarbon" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>450</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12" >
   <property name="geometry" >
    <rect>
     <x>193</x>
     <y>450</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Fraction of organic carbon</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_permeability" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>480</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13" >
   <property name="geometry" >
    <rect>
     <x>211</x>
     <y>480</y>
     <width>132</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Gas permeability [cm2]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14" >
   <property name="geometry" >
    <rect>
     <x>129</x>
     <y>510</y>
     <width>214</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Gas phase dynamic viscosity [g/cm/s]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_viscosity" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>510</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SoilDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SoilDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
